### ISSUE TITLE
panic on macro!(/* comment */)

### ISSUE THREAD

**Issue by lucasem** [rust-lang-nursery/rustfmt#2591]
```rust
fn foo() {
    match 0u32 {
        0 => (),
        _ => unreachable!(/* obviously */),
    }
}
```

```sh
$ rustfmt --version
rustfmt 0.4.1- ( )
$ rustfmt foo.rs
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', libcore/option.rs:335:21
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::print
             at libstd/sys_common/backtrace.rs:71
             at libstd/sys_common/backtrace.rs:59
   2: std::panicking::default_hook::{{closure}}
             at libstd/panicking.rs:207
   3: std::panicking::default_hook
             at libstd/panicking.rs:223
   4: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:402
   5: std::panicking::begin_panic_fmt
             at libstd/panicking.rs:349
   6: rust_begin_unwind
             at libstd/panicking.rs:325
   7: core::panicking::panic_fmt
             at libcore/panicking.rs:72
   8: core::panicking::panic
             at libcore/panicking.rs:51
   9: <rustfmt_nightly::lists::ListItems<'a, I, F1, F2, F3> as core::iter::iterator::Iterator>::next
  10: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::from_iter
  11: <rustfmt_nightly::overflow::Context<'a, T>>::rewrite_items
  12: <rustfmt_nightly::overflow::Context<'a, T>>::rewrite
  13: rustfmt_nightly::overflow::rewrite_with_parens
  14: rustfmt_nightly::macros::rewrite_macro
  15: rustfmt_nightly::expr::format_expr
  16: <rustfmt_nightly::expr::ArmWrapper<'a> as rustfmt_nightly::rewrite::Rewrite>::rewrite
  17: <rustfmt_nightly::lists::ListItems<'a, I, F1, F2, F3> as core::iter::iterator::Iterator>::next
  18: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T, I>>::from_iter
  19: rustfmt_nightly::expr::format_expr
  20: rustfmt_nightly::expr::<impl rustfmt_nightly::rewrite::Rewrite for syntax::ast::Stmt>::rewrite
  21: rustfmt_nightly::visitor::FmtVisitor::walk_stmts
  22: rustfmt_nightly::visitor::FmtVisitor::visit_block
  23: rustfmt_nightly::visitor::FmtVisitor::visit_fn
  24: rustfmt_nightly::visitor::FmtVisitor::visit_item
  25: rustfmt_nightly::reorder::<impl rustfmt_nightly::visitor::FmtVisitor<'a>>::visit_items_with_reordering
  26: rustfmt_nightly::format_input_inner
  27: <scoped_tls::ScopedKey<T>>::set
  28: syntax::with_globals
  29: rustfmt_nightly::run
  30: rustfmt::execute
  31: rustfmt::main
  32: std::rt::lang_start::{{closure}}
  33: std::panicking::try::do_call
             at libstd/rt.rs:59
             at libstd/panicking.rs:306
  34: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:102
  35: std::rt::lang_start_internal
             at libstd/panicking.rs:285
             at libstd/panic.rs:361
             at libstd/rt.rs:58
  36: main
  37: __libc_start_main
  38: _start
```

**Comment by lucasem** [comment]
Another example:

```rust
fn foo() {
    let _ = column!(/* here */);
}
```

**Comment by topecongiro** [comment]
This is fixed on the current master (possibly by #2571). We need a test.
